src/components/LanguageSelection/LanguageSelection.js

```diff
@@ -0,0 +1,104 @@
+import React, { useContext, useState } from "react";
+import { ConfigContext } from "Context";
+import { useAsync } from "react-use";
+import Loader from "components/Loader";
+import { FormattedMessage, useIntl } from "react-intl";
+import { v4 as uuidv4 } from "uuid";
+import {
+  ButtonGroup,
+  Heading,
+  LinkButton,
```

Comment:
For the time being we should use our own `Button` component here with the appropriate variants:

```jsx
<Button variant="anchor" component="button" ... />
```
should result in the same markup, but have the advantage that our own styling layer is applied on top.